public/diagnostic.js

```diff
@@ -1,5 +1,25 @@
 /// Ember Object Diagnostic ///
 
+const Cart = Ember.Object.extend({
+  orders: [],
+  addToCart: function(unitPrice, item, quantity){
+    Cart.orders.push(Order.create({unitPrice: unitPrice, item: item, quantity: quantity }));
+  },
+  totalPrice: Ember.computed(Cart.orders.@each.orders, function(){
+   // code  
+  })
+});
+
+const Order = Cart.extend({
+  orderPrice: Ember.computed('unitPrice', 'quantity', function() {
+    return this.get('unitPrice') * this.get('quantity');
+  })
+});
+
+Cart.addToCart(5,'hats','2');
+Cart.addToCart(20,'desk lamp','1');
+Cart.addToCart(8,'hand towels','3');
```

Comment:
the new cart instance needs to be created, before you can create each order instance.